src/khoj/interface/web/config.html

```diff
@@ -1,70 +1,114 @@
 {% extends "base_config.html" %}
 {% block content %}
-<h2>Content Types</h2>
-<div id="content-configuration">
-    <button onclick="window.location.href='/config/content_type/pdf';">
-        PDF
-    </button>
-    <button onclick="window.location.href='/config/content_type/markdown';">
-        Markdown
-    </button>
-    <button onclick="window.location.href='/config/content_type/org';">
-        Org
-    </button>
-    <button onclick="window.location.href='/config/content_type/ledger';">
-        Ledger
-    </button>
-    <button onclick="window.location.href='/config/content_type/github';">
-        GitHub
-    </button>
+
+<div class="page">
+    <div class="section">
+        <h2 class="section-title">Plugins</h2>
+        <div class="section-cards">
+            <div class="card">
+                <div class="card-title-row">
+                    <img class="card-icon" src="/static/assets/icons/github.svg" alt="Github">
+                    <h3 class="card-title">Github</h3>
+                </div>
+                <div class="card-description-row">
+                <p class="card-description">Set repositories for Khoj to index</p>
+                </div>
+                <div class="card-action-row">
+                    <button class="card-button" onclick="window.location.href='/config/content_type/github';">
+                        Setup
+                        <svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M5 12h14M12 5l7 7-7 7"></path></svg>
+                    </button>
```

Comment:
Could we make this an `anchor` tag? It makes the navigation nicer and allow the user to `ctrl`+`click` and open in a new window. Might make the styling slightly annoying.